pkg/cli/admin/upgrade/status/examples/4.16.0-ec2-control-plane-updated-pdb-prohibits-draining.detailed-output

```diff
@@ -152,6 +146,15 @@ Message: Node build0-gstfj-ci-tests-worker-c-dcz9p is degraded
     nodes: build0-gstfj-ci-tests-worker-c-dcz9p
   Description: failed to drain node: build0-gstfj-ci-tests-worker-c-dcz9p after 1 hour. Please see machine-config-controller logs for more information
 
+Message: Cluster Version version is failing to reconcile (ClusterOperatorDegraded)
+  Since:       0s
```

Comment:
Is this just an artifact of how the mock execution calculates "now" (=last transition of any kind)? If would expect this to be a time since the transition to Failing=True happened.

Reply:
yup, this is because test-now is "the largest `lastTransitionTime` on ClusterVersion conditions", and in the test fixtures, `Failing` is the one with the most recent change-time.  We could add 10m or something when generating "now" if you wanted to see something non-zero here?

Reply:
It's fine, I saw the code doing the right thing after I made this comment and forgot to delete it.